src/ResourceManager/Compute/Commands.Compute/Models/PSVirtualMachine.cs

```diff
@@ -88,34 +91,7 @@ public string ResourceGroupName
         // Gets or sets the storage profile.
         public StorageProfile StorageProfile { get; set; }
 
-        [JsonIgnore]
```

Comment:
@hyonholee Not sure this release is setup to be a breaking change. Can we leave this in the object until next time we have a breaking change release? 

@cormacpayne What do we need to do for a breaking change release?